struct.go

```diff
@@ -18,30 +18,47 @@ type Zone struct {
 
 type Stat struct {
 	Name    string `xml:"name"`
-	Counter int    `xml:"counter"`
+	Counter uint   `xml:"counter"`
 }
 
 type View struct {
 	Name    string `xml:"name"`
-	Zones   []Zone `xml:"zones>zone"`
-	Resstat []Stat `xml:"resstat"`
+	Cache   []Stat `xml:"cache>rrset"`
 	Rdtype  []Stat `xml:"rdtype"`
+	Resstat []Stat `xml:"resstat"`
+	Zones   []Zone `xml:"zones>zone"`
 }
 
 //TODO expand
 type Socket struct {
-	Id           string `xml:"id"`
+	ID           string `xml:"id"`
 	Name         string `xml:"name"`
 	LocalAddress string `xml:"local-address"`
+	References   uint   `xml:"references"`
 }
 
 type Socketmgr struct {
-	References int      `xml:"references"`
```

Comment:
Only in SocketMgr. Its also in the official sample on the BIND website https://kb.isc.org/article/AA-00769/0/Using-BINDs-XML-statistics-channels.html